src/main/java/dev/streamx/cli/publish/payload/PayloadException.java

```diff
@@ -0,0 +1,22 @@
+package dev.streamx.cli.publish.payload;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.io.IOException;
+
+public class PayloadException extends RuntimeException {
+  private PayloadException(String prefix, Exception exception) {
+    super(prefix + exception.getMessage(), exception);
+  }
+
+  public static PayloadException jsonProcessingException(JsonProcessingException exception) {
+    return new PayloadException("Json processing exception. ", exception);
```

Comment:
If this message for user then it should give more details and hint how to proceed in such situation.